conjure-java-jaxrs-client/src/main/java/com/palantir/conjure/java/client/jaxrs/JaxRsClient.java

```diff
@@ -39,7 +39,6 @@ private JaxRsClient() {}
             UserAgent userAgent,
             HostEventsSink hostEventsSink,
             ClientConfiguration config) {
-        // TODO(rfink): Add http-remoting agent as informational
```

Comment:
this is already happening, see com.palantir.conjure.java.okhttp.OkHttpClients#augmentUserAgent